### DISCUSSION TITLE
Welcome to bane Discussions!

### DISCUSSION THREAD

**Discussion by AlaBouali** [AlaBouali/bane#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by underplayer100** [comment]
Is a good tool thanks

**Comment by AlaBouali** [comment]
you are welcome !

**Comment by wyattsss1** [comment]
I need help setting this up I am using windows

**Reply by riccio8** [reply]
what's the problem? btw is a package, u can install it using pip
